src/arch/x86/vm-arch.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
Rename this file to `src/arch/x86/desc.h` which stands for the architecture-specific descriptions.